src/components/Books.css

```diff
@@ -1,10 +1,52 @@
+:root {
+  --black-two: #121212;
+  --white-two: #fff;
+  --white-three: #e8e8e8;
+  --medium-blue: #307bbe;
+  --white: #fafafa;
+  --azure: #0290ff;
+}
+
+body {
+  background-color: var(--white);
+  padding-bottom: 5%;
+}
+
 body > * {
   margin-left: 5%;
 }
 
```

Comment:
| @aamir-asaram Design | Zeplin's Design |
|-|-|
| ![image](https://github.com/aamir-asaram/bookstore/assets/93039359/d6e91686-7e45-4b8e-a98b-4b10cd4d0417) | ![image](https://github.com/aamir-asaram/bookstore/assets/93039359/5bc444de-e1f4-473a-a848-dba494fe6241) |

- [ ] It would be nice if you compare your design with zeplins then you found the below things inconsistent with the given design 🤔

1. It would be great if your app fits on all the computer screens because you can see it is going out of the screen and making it look really unprofessional 😨
2. Kindly make your Navbar background color white to make your it look similar to the Zeplin's design 👍
3. It would be nice if you put the user icon on the left side of your navbar in your design because it is missing in your design 🤔
4. Last but not the least, kindly make your text color and font weight similar to the Zeplin's design to make your app look attractive 😊